### PR TITLE
upgraded great-expectations>=0.15.34 and sqlalchemy>=1.4.44

### PR DESCRIPTION
**Pull request by zhangchi1**
### Upgrade great_expectation py lib version

Hi @denimalpaca,  I realized that our operator is installing `great-expectations 0.13.x ` (published on 3/17/2021) and the latest `great-expectations 0.15.x `has a lot of updates and bug fixes. 
I believe some issues related to `data_asset_name` can be resolved if we keep with the latest version of `great-expectations`
Reference: https://github.com/great-expectations/great_expectations/issues/6260

Let me know if you have any opinions and feel free to let me know what is our best practice for adding test cases.

Look forward to hearing back from you.

Thanks,

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
     url="https://github.com/great-expectations/airflow-provider-great-expectations",
     install_requires=[
         "apache-airflow>=2.1",
-        "great-expectations>=0.13.14",
-        "sqlalchemy>=1.3.16",
+        "great-expectations>=0.15.34",
+        "sqlalchemy>=1.4.44",
```
Comment (denimalpaca) [inline setup.py:21]: Thanks for adding this. I think we should probably pin with an `==` for good measure.

**Review by denimalpaca — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (zhangchi1) [reply setup.py]: Hi @denimalpaca , sorry that I just saw your comment. Updated the install_requires with ==. Btw, do we have auto-squash commits setup in our PR? Or I need to do it on my local? Thank you again ~

Comment (denimalpaca) [reply setup.py]: @zhangchi1 You need to rebase on main now, we just changed `setup.py` to `setup.cfg`. Auto-squash commits are enabled. I'm just doing a little local test on the req updates then I'll approve and @kaxil or I can squash and merge after you rebase. Thank you for contributing!

**Review by denimalpaca — APPROVED** [review]

File: setup.cfg [hunk 2]
```diff
@@ -33,8 +33,8 @@ packages = find_namespace:
 include_package_data = true
 install_requires =
     apache-airflow>=2.1
-    great-expectations>=0.13.14
-    sqlalchemy>=1.3.16
+    great-expectations==0.15.34
+    sqlalchemy==1.4.44
```
Comment (kaxil) [inline setup.cfg:37]:
```suggestion
    great-expectations>=0.15.34
    sqlalchemy>=1.3.16
```

Comment (kaxil) [reply setup.cfg:37]: We should not pin the version -- otherwise it won't work with other libraries. Please keep the upper bound open

**Comment by zhangchi1** [comment]
hi @kaxil I saw the pipeline is running, let me know if we still need a squash?

**Comment by kaxil** [comment]
> hi @kaxil I saw the pipeline is running, let me know if we still need a squash?

All good :)

File: setup.py (hunk 1 above)
Comment (denimalpaca) [reply setup.py]: sorry about suggesting the `==` pinning, didn't realize it would potentially cause problems!

**Pull request merged**